src/Umbraco.Web.UI.Client/src/common/services/varianthelper.service.js

```diff
@@ -0,0 +1,61 @@
+﻿/**
+* @ngdoc service
+* @name umbraco.services.variantHelper
+* @description A helper service for dealing with variants
+**/
+function variantHelper() {
```

Comment:
We are trying to keep away from naming new things `Helper` since these just become buckets of "stuff" that don't make any real sense. Naming is hard but if you can think of a better service name we should try to not use Helper. Maybe

* `contentVariantIdentifier`
* `contentVariantUtility` (I know, not much better than helper ;) 
etc.. ?